test/system/150-login.bats

```diff
@@ -70,9 +70,14 @@ function setup() {
         openssl req -newkey rsa:4096 -nodes -sha256 \
                 -keyout $AUTHDIR/domain.key -x509 -days 2 \
                 -out $AUTHDIR/domain.crt \
-                -subj "/C=US/ST=Foo/L=Bar/O=Red Hat, Inc./CN=localhost"
+                -subj "/C=US/ST=Foo/L=Bar/O=Red Hat, Inc./CN=localhost" \
+                -addext "subjectAltName=DNS:localhost"
     fi
 
+    # Copy a cert to another directory for --cert-dir option tests
+    mkdir -p ${PODMAN_LOGIN_WORKDIR}/client-auth
+    cp $CERT ${PODMAN_LOGIN_WORKDIR}/client-auth
```

Comment:
The `client-auth` name suggests to me that this is trying to authenticate the client; that’s not the case, is it?

Maybe `trusted-registry-cert-dir?`